package.json

```
@@ -25,6 +25,7 @@
     "generate": "nuxt generate",
     "start": "nuxt start",
     "prod": "pnpm build:only && pnpm start",
+    "prod:playwright": "pnpm i18n:copy-test-locales && pnpm build:only && pnpm start",
```

Nit: can shorten this to remove duplication.

```suggestion
    "prod:playwright": "pnpm i18n:copy-test-locales && pnpm prod",
```